src/components/FileList.tsx

```diff
@@ -52,6 +52,17 @@ export interface IFileListProps {
    * The application language translator.
    */
   trans: TranslationBundle;
+
+  /**
+   * Wraps mouse event handler to stop event propagation
+   */
+
+  /**
+   * Wrap mouse event handler to stop event propagation
+   * @param fn Mouse event handler
+   * @returns Mouse event handler that stops event from propagating
+   */
+  stopPropagationWrapper: (fn: React.EventHandler<React.MouseEvent>) => React.EventHandler<React.MouseEvent>;
```

Comment:
It looks that this change is not needed, isn't it?

Reply:
In GitPanel, FileList takes stopPropagationWrapper as a prop.